### DISCUSSION TITLE
How are you using MetalK8s?

### DISCUSSION THREAD

**Discussion by NicolasT** [scality/metalk8s#3085] [General]
We'd love to learn how, for which purposes, where,... MetalK8s is being used by community members! Is this for testing purposes? Running in production? Large clusters or small ones?...

**Comment by NicolasT** [comment]
At [Scality](https://scality.com), we use MetalK8s as part of the [XDM](https://www.scality.com/resources/solution-sheets/ring8-data-sheet/) (*E**X**tended **Data** **M**anagement*) feature of the [RING](https://www.scality.com/products/ring/) product, essentially installing our supported version of the [Zenko](https://zenko.io) open-source multi-cloud data controller project. As such, it gets deployed in many customer environments small and large.

MetalK8s will also form the basis (*platform*) of future Scality products.

**Comment by Baitanik** [comment]
Recently we have tried to bring metalk8s as part of our existing onprem IOT solution. although this integration is very initial stage.. 
we have one IOT platform product , which is used by different iot solutions ... not going into the details of the product.
we have the product up and running in cloud service and customers are using it. 
For the same product existing onprem part was mainly for development purpose.. Developers used to installed in their vms and do development. That was mainly with ansible and kubeadm ... 
Now we are trying to make the onprem solution robust scalable , so that the onprem part can be made enterprise level.. 
That is where metalk8s comes into the picture for us..

**Reply by NicolasT** [reply]
That's a super cool use-case, thanks for sharing :100: If at some point you can tell us which product(s) this applies to, please do :wink:

Happy to hear MetalK8s is useful to you and your team, please make sure to share any issues, potential improvements,... with us, and glad you're part of the community :+1:

**Comment by n1603** [comment]
Our usage of MetalK8s is at initial stage, trying out single node deployment in lab currently. It could be used to deploy some services for device management.

**Reply by NicolasT** [reply]
Another interesting use-case. We frequently perform single-node deployments (in our CI, in testing of the product that includes MetalK8s, in personal clusters,...) so this should be quite stable (assuming enough CPU/memory resources available). Let us know if you run into any issues, or spot potential improvements! We want MetalK8s deployment to be as rock-solid as possible, also for single-node clusters.
